MobileWallet/Screens/Home/TransactionHistory/TransactionsTableViewController.swift

```diff
@@ -48,8 +48,16 @@ protocol TransactionsTableViewDelegate: class {
 
 class TransactionsTableViewController: UITableViewController {
     let CELL_IDENTIFIER = "TransactionTableTableViewCell"
+    let SECTION_HEADER_HEIGHT: CGFloat = 35
```

Comment:
Casing.

Reply:
@kukabi what casing would you use for a constant like this?

Reply:
The guideline (here)[https://swift.org/documentation/api-design-guidelines/#general-conventions] recommends _Names of types and protocols are UpperCamelCase. Everything else is lowerCamelCase._